fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanCOBPartitioner.java

```diff
@@ -58,10 +58,10 @@ public class LoanCOBPartitioner implements Partitioner {
 
     private final Long numberOfDays;
 
-    @Value("#{jobExecutionContext['BusinessDate']}")
+    @Value("#{stepExecutionContext['BusinessDate']}")
     @Setter
     private LocalDate businessDate;
-    @Value("#{jobExecutionContext['IS_CATCH_UP']}")
+    @Value("#{StepExecutionContext['IS_CATCH_UP']}")
```

Comment:
I think the S in the beginning should not be capitalized.

Reply:
Done